docs/csharp/language-reference/operators/assignment-operator.md

```diff
@@ -28,19 +28,17 @@ The following example demonstrates the usage of the assignment operator with a l
 
 :::code language="csharp" source="snippets/shared/AssignmentOperator.cs" id="SnippetSimple":::
 
-The left operand of an assignment receives the *value* of the right hand of the assignment. When the operands are [value types](../builtin-types/value-types.md), the assignment copies the contents of the right-hand operand. When the operands are [reference types](../builtin-types/reference-types.md), assignment copies the reference to the object.
+The left-hand operand of an assignment receives the *value* of the right-hand operand. When the operands are of [value types](../builtin-types/value-types.md), assignment copies the contents of the right-hand operand. When the operands are of [reference types](../builtin-types/reference-types.md), assignment copies the reference to the object.
 
 This is called *value assignment*: the value is assigned.
 
 ## ref assignment
 
-*Ref assignment* assigns the *reference* of its right hand operator to its left hand operand. The left operand is an alias to the right operand. The left operand must be a [ref local](../keywords/ref.md#ref-locals), [ref readonly local](../keywords/ref.md#ref-readonly-locals) or `ref` field in a [`ref struct`]../builtin-types/ref-struct.md). The following example demonstrates the usage of the ref assignment operator:
+*Ref assignment* `= ref` makes its left-hand operand an alias to the right-hand operand. The left-hand operand must be a [ref local](../keywords/ref.md#ref-locals), [ref readonly local](../keywords/ref.md#ref-readonly-locals), or a `ref` field in a [`ref struct`](../builtin-types/ref-struct.md). Both operands must be of the same type.
 
-:::code language="csharp" source="snippets/shared/AssignmentOperator.cs" id="SnippetRefAssignment":::
-
-The `ref` must be applied to the right operands for a ref assignment. Both operands must be of the same type.
+The following example demonstrates the usage of the ref assignment operator:
 
-This is called *ref assignment*: The variable now refers to a different object.
```

Comment:
I didn't understand that sentence, somehow it contradicts the example code.
```csharp
double[] arr = { 0.0, 0.0, 0.0 };
ref double arrayElement = ref arr[0];
```
Here, `arrayElement` becomes an alias to `arr[0]` and they both "refer" to the same (not different) memory location: the first array element.

Or I misinterpreting the sentence?

Reply:
That was poorly written. The concept I wanted to describe is that *ref reassignment* means the variable becomes an alias to a different location. Perhaps using this example would highlight it better:

```csharp
double[] arr = { 0.0, 0.0, 0.0 };
// arrayElement is an alias for arr[0]:
ref double arrayElement = ref arr[0];
// Make arrayElement an alias to arr[1]:
arrayElement = ref arr[1];
```

What do you think?

Reply:
@BillWagner I've added the description of the example. Please check the latest commit.

Reply:
That's perfect.